woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -651,13 +654,8 @@ def test_sets_datetime64_dtype_on_init():
         df = pd.DataFrame(series)
         df.ww.init(logical_types=ltypes)
         assert df.ww.columns[column_name]['logical_type'] == logical_type
-        assert df.ww.columns[column_name]['dtype'] == logical_type.primary_dtype
         assert df[column_name].dtype == logical_type.primary_dtype
 
-
-def test_invalid_dtype_casting():
-    column_name = 'test_series'
```

Comment:
Is there a reason you removed this and combined with the test above?

Reply:
definitely an accident--thanks for catching!